src/components/Spinner/Spinner.js

```diff
@@ -5,22 +5,36 @@ import cx from 'classnames';
 import { flavorable } from '../../decorators';
 
 export const Spinner = (props) => {
-  const { className, size, ...rest } = props;
+  const { className, size, container, fixed, ...rest } = props;
```

Comment:
Wouldn't it be better to have a `<SpinnerContainer />` component that accepts an additional className etc? A use-case for that would be different heights set via a custom class. Also, the `fixed` prop is only relevant if container is set, which is a bit confusing imo. I'm in favor of splitting this into another component